pkg/scheduler/framework/runtime/framework.go

```diff
@@ -608,7 +623,15 @@ func (f *frameworkImpl) RunPreFilterPlugins(ctx context.Context, state *framewor
 	var result *framework.PreFilterResult
 	var pluginsWithNodes []string
 	skipPlugins := sets.New[string]()
+	logger := klog.FromContext(ctx)
+	logger = klog.LoggerWithName(logger, "PreFilter")
+	// TODO(knelasevero): Remove duplicated keys from log entry calls
+	// When contextualized logging hits GA
+	// https://github.com/kubernetes/kubernetes/issues/111672
+	logger = klog.LoggerWithValues(logger, "pod", klog.KObj(pod))
```

Comment:
The pod should probably be added by the caller of RunPreFilterPlugins (the scheduler package)

Reply:
You mean add in here?
pkg/scheduler/schedule_one.go:398
```
         logger = klog.LoggerWithValues(logger, "pod", klog.KObj(pod))
	// Run "prefilter" plugins.
	preRes, s := fwk.RunPreFilterPlugins(ctx, state, pod)
```

Reply:
Yes, somewhere in there, but I get your point about the feature gate.